cfme/fixtures/vm_console.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just for my understanding, why make this fixture code as opposed to vm_console class code?

Reply:
Never mind they have always been fixtures...but I don't remember why (-;

Reply:
I believe this is because these are pre-requisites for the test and vm_console class is instantiated much later in the test.